spec/rubocop/cop/style/hash_syntax_spec.rb

```diff
@@ -406,6 +428,17 @@
         expect_no_offenses('{ :a => :b, :c => :d }')
       end
     end
+
+    it 'registers an offense for implicit `call` message' do
```

Comment:
Isn't "explicit" correct instead of "implicit"?

```suggestion
    it 'registers an offense for explicit `call` method' do
```

Reply:
Isn't "explicit `call`" is when you do `.call(...)` 🙂

Reply:
If that’s the case, it seems that the test code and the description are not aligned. If it’s "implicit", the test should be for `.(a: 0)` instead of `.call(a: 0)`.